plugin/clang.vim

```diff
@@ -841,13 +857,57 @@ func! s:ClangCompleteInit(force)
 endf
 "}}}
 "{{{ ClangExecuteNeoJobHandler
-"handles event: exit
+"Handles events: stdout, stderr, exit
+"Tmps in memory
 func! ClangExecuteNeoJobHandler(job_id, data, event)
-  if a:event == 'exit'
-    call ClangExecuteDone(self.fstdout, self.fstderr)
+  if a:event == 'stdout'
+    call s:PDebug("ClangExecuteNeoJobHandler", "stdout", 2)
+    if len(b:clang_memtmps[0]) != 0
```

Comment:
I think you can use the same buffer var `b:clang_state`, you can set `stdout` and `stderr` in this dict.